docs/api/browser-view.md

```diff
@@ -70,5 +70,29 @@ The `bounds` of this BrowserView instance as `Object`.
 
 #### `view.setBackgroundColor(color)` _Experimental_
 
-* `color` string - Color in `#aarrggbb` or `#argb` form. The alpha channel is
```

Comment:
Is it true that this used to take alpha as the first component instead of the last? Is this PR changing the meaning of `view.setBackgroundColor('#ff7f7f7f')`?

Or was this old documentation wrong?

Reply:
@nornagon we did, and this does change the behavior. 

I think personally speaking that this merits a breaking change but if we'd prefer to keep the change surface smaller I can manipulate it prior to passing it to `ParseCSSColor` so it stays consistent with previous behavior.

Reply:
Oof, that's unfortunate. I'm not sure it's really OK to break this API without warning. Despite the fact that it's marked experimental, it's presumably been this way since it was introduced in 2017, and the BrowserWindow version even longer. The result of this change would be that people's windows randomly changed color and transparency, which is bemusing at best.

If we're going to make this break, it should come with some backwards-compatibility mechanism like a property name change; something so that people's apps don't break without warning.

Reply:
@nornagon fair, and i agree. i can add a case to catch for AARRGGBB and make this semver-minor. I'd still like to consider deprecating it in the future.

Reply:
addressed!